src/django/pfb_analysis/management/commands/analysis_batch_create.py

```diff
@@ -0,0 +1,109 @@
+    help = """ Create a new AnalysisBatch based on the input shapefile
+
+    A new analysis job is created for each Polygon/MultiPolygon feature in the shapefile.
+    Each feature must have the following attributes:
+    - Name: The name of the boundary
+    - State: The 2-letter state abbreviation code that the boundary is contained within
+
+    If --submit is provided, the jobs are automatically submitted after creation
+
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('shapefile_url', type=str)
+        parser.add_argument('--user', default='systems+pfb@azavea.com', type=str)
+        parser.add_argument('--submit', action='store_true',
+                            help='Automatically submit jobs after creation')
+
+    def handle(self, *args, **options):
+        shapefile_url = options['shapefile_url']
+
+        User = get_user_model()
+        user = User.objects.get(email=options['user'])
+
+        batch = AnalysisBatch.objects.create(created_by=user, modified_by=user)
+
+        try:
+            tmpdir = tempfile.mkdtemp()
+            self.stdout.write('Using temp dir: {}'.format(tmpdir))
+
+            # Extract download zipfile and find shp filename
+            local_zipfile = os.path.join(tmpdir, 'boundary.zip')
+            download_file(shapefile_url, local_zipfile)
+            with zipfile.ZipFile(local_zipfile) as zip:
+                files = zip.namelist()
+                zip.extractall(tmpdir)
+                local_shapefile = next(filename for filename in files if filename.endswith('.shp'))
+            local_shapefile = os.path.join(tmpdir, local_shapefile)
+
+            # Open shp and trigger analysis job for each feature in layer
+            self.stdout.write('Opening shapefile: {}'.format(local_shapefile))
+            with fiona.open(local_shapefile, 'r') as source:
+                for feature in source:
+                    city = feature['properties']['city']
+                    state = feature['properties']['state']
+                    label = '{}, {}'.format(city, state)
+                    name = Neighborhood.name_for_label(label)
+
+                    # Get or create neighborhood for feature
+                    neighborhoods = Neighborhood.objects.filter(organization=user.organization,
+                                                                name=name)
+                    try:
+                        neighborhood = neighborhoods[0]
+                    except IndexError:
+                        geom = GEOSGeometry(json.dumps(feature['geometry']))
+                        neighborhood = Neighborhood.objects.create_from_geom(geom, **{
+                            'label': '{}, {}'.format(city, state),
+                            'state_abbrev': state,
+                            'organization': user.organization,
+                            'created_by': user,
+                            'modified_by': user,
+                        })
```

Comment:
Trying to figure out what this gets over a `get_or_create` followed by `if created` setup led to discussion about Neighborhood mutability.  Per that discussion, this behavior, where changes to a boundary are ignored if a neighborhood with that name already exists, is not what we want.  For now, we should overwrite the shapefile every time.  In the future we might want to detect changes and break the boundary out into a related model so we can update a neighborhood boundary but maintain the ability to know about/recreate older versions.